dist/bearhug-angular.js

```diff
@@ -0,0 +1,207 @@
+(function(angular) {
+    bearhugAuthenticator.$inject = ["$http", "$injector", "$q", "bearhugStorage"];
+    bearhugInterceptor.$inject = ["$q", "$injector", "bearhugStorage"];
+    bearhugStorage.$inject = ["bearerUtils"];
+    interceptorFilter.$inject = ["$q"];
+    angular.module("talis.bearhug", []);
+    angular.module("talis.bearhug").factory("bearerUtils", bearerUtils);
+    function bearerUtils() {
+        var BEARER_REGEX = /^Bearer (.*)$/;
```

Comment:
@junglebarry @markwallsgrove according to the RFC for bearer tokens in the oauth spec 
https://tools.ietf.org/html/rfc6749#section-8.1  the token can only contain 
```
"-" / "." / "_" / DIGIT / ALPHA
```

So I think this regex would be correct

```
/^Bearer ([a-zA-Z0-9\._\-]+)$/?
```